Samples/Caboodle.Samples/View/PhoneDialerPage.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
```

Comment:
We're using `views:BasePage` elsewhere in the sample.  I think we should be consistent and switch to it here too...